lessonbuilder/tool/src/webapp/templates/instructions/website.html

```diff
@@ -25,7 +25,7 @@
 <h3> ADDING CONTENT FROM PACKAGES SUCH AS CAMTASIA, WIMBA CREATE AND ARTICULATE</h3>
 
 <p>
-The "Upload Content in ZIP File" dialog allows you to add content from a file.
+The "Upload Content from ZIP File" dialog allows you to add content from a file.
```

Comment:
Should be "Update Content from a ZIP File" (the extra a).

Reply:
Good catch, @buckett! Thanks. Yes, it should match the message above. So it should be:
 
```suggestion
The "Upload Content from a ZIP File" dialog allows you to add content from a file.
```

Reply:
Should I submit a new PR for this same Jira issue? Or create a new Jira issue for this change?

Reply:
As per the recommendation of the Core Team on December 11th's call, I have submitted another PR for SAK-41016 to add the "a". Thanks for spotting this, @buckett!